releases/release-6.1.3.md

```diff
@@ -16,7 +16,7 @@ Quick access: [Quick start](https://docs.pingcap.com/tidb/v6.1/quick-start-with-
 
     - TiCDC
 
-        - Change the default value of [`transaction-atomicity`](/ticdc/ticdc-sink-to-mysql.md#configure-sink-uri-for-mysql-or-tidb) from `table` to `none` to improve stability [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
+        - Change the default value of [`transaction-atomicity`](/ticdc/ticdc-sink-to-mysql.md#configure-sink-uri-for-mysql-or-tidb) from `table` to `none`, which helps reduce replication latency and mitigate OOM risks, and ensures that only a few transactions are split, instead of all transactions [#7505](https://github.com/pingcap/tiflow/issues/7505) @[asddongmen](https://github.com/asddongmen)
```

Comment:
please add 1024

Reply:
Done in [0fb6ed1](https://github.com/pingcap/docs/pull/11659/commits/0fb6ed1ffcf711e05f8e0802dcf6843dff096b45)